bool.go

```diff
@@ -0,0 +1,90 @@
+// Code generated by github.com/tarantool/go-option; DO NOT EDIT.
+
+package option
+
+import (
+	"github.com/vmihailenco/msgpack/v5"
+	"github.com/vmihailenco/msgpack/v5/msgpcode"
+)
+
+type Bool struct {
+	value  bool
+	exists bool
+}
+
+func SomeBool(value bool) Bool {
+	return Bool{
+		value:  value,
+		exists: true,
+	}
+}
+
+func NoneBool() Bool {
+	return Bool{
+		exists: false,
+		value:  zero[bool](),
+	}
+}
+
+func (o Bool) IsSome() bool {
+	return o.exists
+}
```

Comment:
Please add comments to the public API methods and types. Despite the fact that it will be generated code and comments generated, but this is a general rule in our repositories.
```suggestion

// Bool is ...
type Bool struct {
	value  bool
	exists bool
}

// SomeBool returns...
func SomeBool(value bool) Bool {
	return Bool{
		value:  value,
		exists: true,
	}
}

// NoneBool returns ...
func NoneBool() Bool {
	return Bool{
		exists: false,
		value:  zero[bool](),
	}
}

// IsSome returns ...
func (o Bool) IsSome() bool {
	return o.exists
}
```